src/ingress.ts

```diff
@@ -40,6 +40,11 @@ export interface IngressProps extends base.ResourceProps {
    * extension, if the ingress controller fulfilling the ingress supports SNI.
    */
   readonly tls?: IngressTls[];
+
+  /**
+   * Ingress Class Name is an optional String property.
+   */
+  readonly ingressClassName?: string;
```

Comment:
"ingress" is already implied 

```suggestion
  readonly className?: string;
```